Seminar_02/task_02.py

```diff
@@ -0,0 +1,9 @@
+# 2 – Напишите программу, которая принимает на вход число N и выдает набор произведений чисел
+# от 1 до N.
+# Пример:
+# пусть N = 4, тогда [ 1, 2, 6, 24 ] (1, 1*2, 1*2*3, 1*2*3*4)
+
+N = int(input('N: '))
+fact = [1]
+[fact.append(fact[i] * (i + 2)) for i in range(N - 1)]
```

Comment:
Можно по разному. В одну строку вряд ли получиться. Самый простой вариант так:
```suggestion
fact = []
last = 1
for i in range(1, N + 1):
    last *= i
    fact.append(last)
```
Этот же вариант можно переписать и в строчку, но только с использованием моржового оператора.
```suggestion
last = 1
fact= [last := last * i for i in range(1, N + 1)]
```